frontend-react/src/components/Admin/OrgsTable.tsx

```diff
@@ -30,12 +35,21 @@ export function OrgsTable() {
     const { memberships } = useSessionContext();
 
     const handleSelectOrgClick = (orgName: string) => {
+        const {
+            state: { active: { senderName, memberType } = {} },
```

Comment:
That's a pretty confusing destructure; anything against `state.active.memberType`?

Reply:
it's more that I like destructuring as a way of declaring in one place at the top of a function what sort of data is needed for the function, and what pieces of data we might need from a complex object. Also like the ability to pluck out multiple pieces of data from a complex object in one go for easy reference later. In this case it's a short function where each variable is only referenced once so it's pretty much a toss up that would come down to stylistic preference, especially if we can use optional chaining.